bin/oref0-setup.sh

```diff
@@ -200,7 +204,7 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
         echo
     fi
 
-    if grep -qa "Explorer HAT" /proc/device-tree/hat/product 2>/dev/null ; then
+    if grep -qa "Explorer HAT" /proc/device-tree/hat/product &>/dev/null ; then
```

Comment:
The -q flag already squashes stdout.  The 2>/dev/null was squashing stderr (file does not exist) as well.  Any reason to change that?